src/List/Dl.php

```
@@ -33,41 +33,71 @@ final class Dl extends BaseBlock
+     * );
      * ```
+     *
+     * @param array{0: Dt|string|Stringable, 1: Dd|string|Stringable} ...$pairs Arrays of `[term, description]` pairs.
+     *
+     * @return static New instance with the appended term-description pairs.
      */
-    public function dt(string|Stringable $content): static
+    public function terms(array ...$pairs): static
     {
-        $dt = Dt::tag()->content($content);
+        $dl = $this;
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $dl. Configured minimum length is 3. (undefined)

(ShortVariable)